core/identity/src/cli/identity.rs

```diff
@@ -264,6 +274,34 @@ impl IdentityCommand {
                         .map_err(|e| anyhow::Error::msg(e))?,
                 )
             }
+            IdentityCommand::Export {
+                node_or_alias,
+                file_path,
+            } => {
+                let node_id = node_or_alias.clone().unwrap_or_default().resolve().await?;
+                let key_file = bus::service(identity::BUS_ID)
+                    .send(identity::GetKeyFile(node_id))
+                    .await?
+                    .map_err(|e| anyhow::Error::msg(e))?;
+
+                match file_path {
+                    Some(file) => {
+                        if file.parent().map_or(false, |parent| !parent.is_dir()) {
+                            anyhow::bail!(
+                                "Directory '{:?}' does not exists or is write protected",
```

Comment:
`Some()` in the error message is confusing and ugly. That's because `parent()` returns `Option<PathBuf>`. I believe there should be a `match` here.
```
Error: Directory 'Some(".....")' does not exists or is write protected
```

Reply:
I dropped directory checking entirely so we fallback to whatever `fs::write()` will return.